package.json

```diff
@@ -21,23 +22,15 @@
     "ts-jest": "^22.4.2",
     "typescript": "2.8.1"
   },
-  "keywords:": [
-    "runtime",
-    "type",
-    "validation",
-    "typescript"
-  ],
+  "keywords:": ["runtime", "type", "validation", "typescript"],
   "repository": {
     "type": "git",
     "url": "https://github.com/pelotom/runtypes"
   },
   "jest": {
     "verbose": false,
     "testRegex": ".*/*.spec.ts$",
-    "moduleFileExtensions": [
-      "js",
-      "ts"
-    ],
+    "moduleFileExtensions": ["js", "ts"],
```

Comment:
Can you revert this file? I need to add `package.json` to `.prettierignore`.

Reply:
May I add that?

Reply:
I'll take care of it :)